### PR TITLE
Clarify application versions

### PR DESCRIPTION
**Pull request by keirthana**
When updating an application, clarify wording around application versions being available and accessible to users for launch and use.

### CODE REVIEW

File: .wordlist.txt [hunk 1]
```diff
@@ -124,6 +124,7 @@ iptables
 init
 interprocess
 Javascript
+JIRA
```
Comment (ajanon) [inline .wordlist.txt:127]: What is this for?

File: howto/application/update.md [hunk 2]
```diff
@@ -84,7 +84,7 @@ In addition, each version has a status which indicates the status of the bootstr
 <a name="publish-application-versions"></a>
 ## Publish application versions
 
-The most important part of an application version is the `published` field. If a version is marked as published, it is accessible to users of Anbox Cloud. Generally when launching containers by using the AMS REST API, if no specific application version is given, the last published version of an application is used to create the container.
+The most important part of an application version is the `published` field. If a version is marked as published, it is available to launch and use. Generally when launching containers by using the AMS REST API, if no specific application version is given, by default, the last published version of an application is used to create the container.
```
Comment (ajanon) [inline howto/application/update.md:87]: Shouldn't it be 'latest'?
```suggestion
The most important part of an application version is the `published` field. If a version is marked as published, it is available to launch and use. Generally when launching containers by using the AMS REST API, if no specific application version is given, by default, the latest published version of an application is used to create the container.
```

What do you think?

**Review by adglkh — APPROVED** [review]
Regarding the term `last` vs `latest`, not sure which one we all agree on in the end. I am fine with either.

File: .wordlist.txt (hunk 1 above)
Comment (keirthana) [reply .wordlist.txt:127]: Oops, I think this was a change meant for the other release notes PR and it crept it here. I will revert this in this PR as it is out of context.

File: howto/application/update.md (hunk 2 above)
Comment (keirthana) [reply howto/application/update.md:87]: Both are fine but in the context of versions, I would agree that 'latest' can be favoured.

**Pull request merged**